src/main/java/uk/gov/laa/ccms/caab/service/ApplicationService.java

```diff
@@ -1385,4 +1543,69 @@ private String findAwardType(
         .orElseThrow(() -> new CaabApplicationException(
             String.format("Failed to find AwardType with code: %s", award.getAwardType())));
   }
+
+  /**
+   * Prepares the proceeding summary for a specific application.
+   *
+   * @param id The ID of the application.
+   * @param application The application details.
+   * @param user The user performing the operation.
+   */
+  public void prepareProceedingSummary(
+      final String id,
+      final ApplicationDetail application,
+      final UserDetail user) {
+
+    getDefaultCostLimitation(application);
+
+    if (!application.getAmendment()
```

Comment:
Now that it's a Boolean, does this need to be Boolean.TRUE.equals(application.getAmendment()) to protect against NullPointerException?

Reply:
Amended